### PR TITLE
increase acc

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Fixes #2029. See https://github.com/PyTorchLightning/pytorch-lightning/pull/2029#discussion_r433296557

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/base/utils.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def assert_speed_parity(pl_times, pt_times, num_epochs):
         f"lightning was slower than PT (threshold {max_diff_per_epoch})"
 
 
-def run_model_test_without_loggers(trainer_options, model, min_acc=0.45):
+def run_model_test_without_loggers(trainer_options, model, min_acc=0.50):
```
Comment (williamFalcon) [inline tests/base/utils.py:28]: some tests specifically set their own

**Comment by Borda** [comment]
seems we have a hanging test for macOS and py3.8
https://github.com/PyTorchLightning/pytorch-lightning/pull/2044/checks?check_run_id=728728190
@williamFalcon it is already from master

**Comment by Borda** [comment]
@SkafteNicki this test is causing problem on macOS and py3.8
`tests/loggers/test_all.py::test_logger_reset_correctly[Batch-size-Finder]`
it is hanging there infinitely long and then the test is timeouted... any idea why?

**Comment by SkafteNicki** [comment]
@Borda not completly sure why it is failing specific on mac. Do you know how much RAM the machines have the test are running on? Since the test currently are evaluated on cpu, it is only when the RAM is filled up we get an OOM error and the search stops. Maybe it would help switching these tests to gpu, since the memory is smaller the test will be faster?

**Comment by Borda** [comment]
@SkafteNicki it is running on the very same servers, just different images... 
also, it is failing on just one particular configuration...

**Comment by SkafteNicki** [comment]
@Borda am I reading the checks correct, that it works on mac with python 3.7 but not with python 3.8?

File: setup.cfg [hunk 2]
```diff
@@ -6,12 +6,16 @@ norecursedirs =
 python_files =
     test_*.py
 # doctest_plus = disabled
-addopts = --strict
+addopts =
+    --strict
+    --doctest-modules
+    --durations=0
```
Comment (awaelchli) [inline setup.cfg:12]: does it mean these options are added to every py.test command? 
what does durations=0 mean?

Comment (Borda) [reply setup.cfg:12]: yes, it does

**Comment by Borda** [comment]
> @Borda am I reading the checks correct, that it works on mac with python 3.7 but not with python 3.8?

exactly... :]

**Comment by Borda** [comment]
This solves the issue with invalid accuracy tests...
